docs/community/otherSoftware.md

```diff
@@ -0,0 +1,80 @@
+nicely with Xournal++.
+
+## Xournal
+
+Xournal++ started as a C++ rewrite of Xournal by [Denis Aroux](http://people.math.harvard.edu/~auroux/), which is programmed in C and hosted on sourceforge.
+Xournal++ is actively developed, while Xournal has stopped developing new features after mid 2016. Denis Aroux notes on the [Xournal website]((http://xournal.sourceforge.net/)):
+
+> You may also want to try out Xournal++ (which started out as a rewrite of Xournal in C++, but now has evolved well beyond the original, while retaining a decent amount of compatibility).
+
+Xournal++ still uses some parts of Xournal's code, most notably Xournal's shape recogntion code. The two applications share a similar look. Xournal++ contains most of Xournal's functionality,
+while adding some useful pluses.
+Xournal++ can open Xournal .xoj-documents and export to it in compatibility mode. We are greatly thankful to Denis Aroux for his wonderful software and all of Xournal's contributors.
+
+The following lists highlight the differences between the latest development version Xournal++ {{ version.latest_unstable }} and Xournal 0.4.8.2016.
+
+### Features available in Xournal++, but not in Xournal
+
+- LaTeX tool
+- Audio recording and playback
+- Lua plugins
+- Pen pressure support
+- Sidebar with page and layer previews
+- Page layout in multiple rows/columns
+- Grid and rotation snapping
+- Select object tool
+- Rectangle, Circle, Arrow, Coordinate system and Spline drawing types
+- Flexible remapping of input devices
+- Zoom gestures
+- Drawing modifiers by drawing direction
+- Export to .png and .svg, and various export options
+- Periodic autosave and auto-backup tool
+- Staves and Isodotted page backgrounds
+- More cursor icon options
+- Floating toolbox
+- Configurable toolbars
+- Language selectable in the GUI
+- Option to easily append new pdf pages
+- More distribution options on Linux (PPA, flatpak, appimage and snap packages)
+- Translations in more languages
+
+### Features available in Xournal, but not in Xournal++
+
+- Input via Xinput (which gives better strokes on some devices)
+- One page mode (useful in presentations)
+- Mirror images of strokes via the selection tool
+- A complete [documentation](http://xournal.sourceforge.net/manual.html) and a [well documented file format](http://xournal.sourceforge.net/manual.html#file-format)
+- Inlusion in the official Debian repository
+
+### Known issues/bugs/inconveniences of Xournal, resolved in Xournal++
+
+TODO
+
+### Performance comparision between Xournal++ and Xournal
+
+- On most systems Xournal++ has significantly reduced memory usage compared to Xournal
+- Xournal++ uses PDF caching for faster display of PDF pages
+- Xournal++ has code to detect memory leaks, which Xournal does not
+- Xournal++'s .xopp files are usually lower in file size thanks to reducing unnessarily high precision of stroke coordinates
```

Comment:
Do you know if these claims are still true?

Reply:
I am not sure. I think it would be good to make some fresh performance measurements comparing the two apps.

Reply:
Hmm, these claims seem on thin ice. I rather delete them. In particular the last one is plain false. Xournal only stores two decimal places. Moreover I have the impression that something must be wrong with pdf caching in Xournal++. After switching from a large pdf to a new document I couldn't see the memory consumption drop to a reasonable level (comparable to after startup).

Reply:
I have removed the unbased claims and will merge now.